stingray/events.py

```diff
@@ -0,0 +1,212 @@
+"""
+Definition of :class:`EventList`.
+
+:class:`EventList` is used to handle photon arrival times.
+"""
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from stingray.simulator.events import events_to_lc, lc_to_events, assign_energies
+
+from .lightcurve import Lightcurve
+import stingray.io as io
+import stingray.utils as utils
+
+import numpy as np
+
+
+class EventList(object):
+    def __init__(self, times, energies=None, mjdref=0, dt=0, notes="", gti=None, pi=None,
+                 pha=None):
+        """
+        Make an event list object from an array of time stamps
+
+        Parameters
+        ----------
+        times: iterable
+            A list or array of time stamps
+
+        Other Parameters
+        ----------------
+        dt: float
+            The time resolution of the events. Only relevant when using events
+            to produce light curves with similar bin time.
+
+        energies: iterable
+            A list of array of photon energy values
+
+        mjdref : float
+            The MJD used as a reference for the time array.
+
+        gtis: [[gti0_0, gti0_1], [gti1_0, gti1_1], ...]
+            Good Time Intervals
+
+        pi : integer, numpy.ndarray
+            PI channels
+
+        pha : float, numpy.ndarray
+            Photon energies
+
+        Attributes
+        ----------
+        time: numpy.ndarray
+            The array of event arrival times, in seconds from the reference
+            MJD (self.mjdref)
+
```

Comment:
Hmm, that's a good point. I don't mind either way, as long as we're consistent.

While we're talking about consistency. I noticed that `Simulator` has a method `periodogram`. Could we perhaps rename that to `powerspectrum`? We discussed a few months ago whether we should call the squared amplitude of the Fourier transform power spectrum or periodogram, and I think we decided to use power spectrum because it's more commonly used. It'd be nice to be consistent about this everywhere.